config/redirects.txt

```diff
@@ -13,3 +13,6 @@
 ^\/getting-started -> /raspberrypi/nodejs/getting-started
 
 ^\/configuration\/resin-base-images(\/|$) -> /runtime/resin-base-images
+
+# Renamed pages
+^\/deployment\/wifi\/ -> /deployment/network/
```

Comment:
could be 

`^\/deployment\/wifi(\/.*)? -> /deployment/network/$1` to keep the rest of the URL